iphone/Maps/LocalizedStrings/ar.lproj/Localizable.strings

```diff
@@ -1392,6 +1392,9 @@
 /* Title for the "Recover All" button on the Recently Deleted Lists screen. */
 "recover_all" = "استرداد الكل";
 
+/* Сообщение для показа ошибки отпутствия приложения для выбора файлов в файловой системе телефона */
+"error_no_file_manager_app" = "Unable to open the file explorer because no suitable app is installed on your device. Please install a file manager from an app store and try again.";
```

Comment:
We separate translations commits like this:
1. A commit with strings.txt change and optionally with related code change.
2. A separate strings regeneration commit (that can be removed easily during rebase if necessary, to avoid merge conflicts), it creates necessary resource files for iOS and Android.
3. As this is android-only translation, iOS strings should not be generated.